Correct code coverage

thom-at-redhat opened the pull request.

thom-at-redhat committed:
Correct code coverage

thom-at-redhat committed:
Updated makefile targets

thom-at-redhat committed:
Merge branch 'devel' into correct_code_coverage

AaronH88 commented on Makefile:
```diff
@@ -49,6 +49,66 @@ endif
+OS=linux
+STABLE_KIND_VERSION=v0.20.0
+
+kind:
```
Why add kind as part of our makefile?

thom-at-redhat replied:
Per discussions, removed

AaronH88 commented on Makefile:
```diff
@@ -86,6 +146,19 @@ else
+COVERAGE_FILE='coverage.txt'
+
+coverage: build-all kind
+	PATH="${PWD}:${PATH}" go test $$(go list ./... | grep -v $(BLOCKLIST)) \
```
the indentation you picked in this PR dosent match the rest of the makefile, is their a reason for this?

thom-at-redhat replied:
Made it match

AaronH88 commented on Makefile:
```diff
@@ -49,6 +49,66 @@ endif
 receptor: $(shell find pkg -type f -name '*.go') ./cmd/receptor-cl/receptor.go
 	CGO_ENABLED=0 GOFLAGS="-buildvcs=false" go build -o receptor $(DEBUGFLAGS) -ldflags "-X 'github.com/ansible/receptor/internal/version.Version=$(VERSION)'" $(TAGPARAM) ./cmd/receptor-cl
 
+clean:
```
why move this?

thom-at-redhat replied:
alphabetizing so we can find targets faster.  We need to reorder all of them in a future PR

AaronH88 commented:
This PR also added kind instillation, but this isnt reflected in either the commit message or PR message. 
If adding kind instillation via a makefile is something we want to do (im not convinced) then please adjust these messages appropriately

thom-at-redhat committed:
Address review comments

AaronH88 commented on Makefile:
```diff
@@ -49,6 +49,40 @@ endif
+	@rm -fv receptor-python-worker/dist/*
+	@rm -rfv receptorctl-test-venv/
+
+ARCH=amd64
```
I think we can use the `uname` command here to discover the os and arch: https://stackoverflow.com/a/714135

thom-at-redhat replied:
we could but it's more involved since the strings don't match.  As we discussed, I would strongly prefer doing such a change in a separate PR

thom-at-redhat commented:
> This PR also added kind instillation, but this isnt reflected in either the commit message or PR message. If adding kind instillation via a makefile is something we want to do (im not convinced) then please adjust these messages appropriately

Removed after discussions

AaronH88 reviewed (approved):
lgtm
